src/gotcha_dl.c

```diff
@@ -63,11 +65,11 @@ static void* dlsym_wrapper(void* handle, const char* symbol_name){
      return binding->user_binding->wrapper_pointer;
 }
 
+struct gotcha_binding_t dl_binds[] = {
```

Comment:
We were getting away with allocating these things on the stack, they only pointed to data we could easily discard. I'm worried user code might have gotten away with the same thing